### ISSUE TITLE
Drag & drop, Ctrl+Shift+arrows, Ctrl+Shift+PageUp/PageDown, ...

### ISSUE THREAD

**Issue by dumblob** [gristlabs/grist-core#548]
Any plans to implement more interactivity resembling Excel basics - e.g. drag & drop of selected cells, selection using keyboard (e.g. Ctrl+Shift+Up), etc.?

I find myself lacking this interactivity a lot. For my use cases this degrades Grist to a database tool rather than being the go-to daily driver for (relational) tables.

**Comment by paulfitz** [comment]
Shift-Up works, adding Ctrl-Shift-Up seems a reasonable addition to me, I'd say we'd accept a PR for that - what do you think @dsagal?

If you select some cells, you can "drop" them somewhere using the copy and paste keyboard shortcuts. Not quite drag & drop I agree. I don't think we have specific plans to add this, but we do continually polish core ergonomics and this is good feedback, thanks.

**Comment by dsagal** [comment]
Agreed, adding Ctrl-Shift-(Arrow Keys) would be great, and should not be hard to add.

By the way, the list of keyboard shortcuts supported by Grist is available at https://support.getgrist.com/keyboard-shortcuts/#navigation (and using `F1` or `Ctrl + /` in the product), plus the system standard cut/copy/paste shortcuts are supported.

As for dragging, Grist supports dragging of rows (in unsorted views), and of columns. These are the far more common needs in a database, compared to dragging cells or ranges between records. Cut/copy/paste shortcuts work for cells and ranges.
